rusty_pizza_server/src/order_model/order.rs

```diff
@@ -169,6 +169,21 @@ impl Order {
             })
         }
     }
+
+    pub fn user_has_to_pay(self, user: u32) -> Option<Money> {
```

Comment:
I'm uncertain whether we really need this method. To be precise: I think I would not call it. If I'm working on a user level I would first get the `Meals` for the user and then calculate the total price on that rather than calling this method because it feels more natural to me. Thus I would prefer removing this method.

If you want to keep it:
How about `get_charge_for_user` / `get_price_for_user`? Although you can do funny things with the current  name...:laughing: 

```rust
let the: Order = ...;
let _ = the.user_has_to_pay(user_id);
```

Joking aside, please document why this returns an `Option`.

Reply:
On the one hand I agree with you, we do not really need this method, but on the other Hand I think it does not really hurt. You got an direct way, to get the single bills for an user of an order.

How about renaming it to `get_bill_of_user`